eks-distro-base/Dockerfile.minimal-nonroot

```diff
@@ -0,0 +1,18 @@
+# Copyright 2020 Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+ARG BASE_IMAGE
+FROM ${BASE_IMAGE} 
+
```

Comment:
Should we `chown 65532:65532 /home/nonroot`?

Reply:
good question.. let me look at what distroless does for that

Reply:
couldnt find where in distroless they set this, but Im thinking you are right so added it to the base dockerfile.